.github/workflows/ci-semantic-pull-request.yml

```
@@ -24,7 +24,6 @@ on:
       - opened
       - reopened
       - edited
-      - synchronize
 
```

```suggestion

concurrency:
  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event.number }}
  cancel-in-progress: true

```

However, it's possible to squash pending runs.